lib/zendesk_apps_support/package.rb

```diff
@@ -31,15 +31,19 @@ def initialize(dir, is_cached = true)
       @warnings = []
     end
 
-    def validate(marketplace: true, skip_marketplace_translations: false, error_on_password_parameter: false)
+    def validate(marketplace: true, skip_marketplace_translations: false, error_on_password_parameter: false,
+                 enable_custom_objects_v2_validation: false)
```

Comment:
nit: how about renaming the arg `validate_custom_objects_v2_requirements` or `validate_custom_objects_v2` to be shorter?

Reply:
I'd recommend avoiding acronyms when possible. `validate_custom_objects_v2` seems a better option IMHO.